Project/zepeto-modules/Assets/Zepeto ScreenShot Module/ZepetoScript/ScreenShotModeManager.ts

```diff
@@ -118,11 +119,14 @@ export default class ScreenShotModeManager extends ZepetoScriptBehaviour {
         // Enable Selfie Camera
         this.SetSelfieCameraActive(true);
         // Enabling IKPass for Selfie Pose Settings
-        this.SetIKPassActive(true); 
+        this.SetIKActive(true);
+        // Disable Moveturn Animation
+        this.SetMoveturnActive(false);
+        //Add speed
+        this.AddRunSpeed(this.additionalRunSpeedInSelfieMode);
+        this.AddWalkSpeed(this.additionalWalkSpeedInSelfieMode);
```

Comment:
These settings remain as we close the screenshot UI since we do not reset the screenshot-specific settings on closing.

We must clean up and restore our settings before getting into the screenshot UI.

Of course, the same thing applies to the default screenshot mode.

Reply:
I've recognized there's an initialization issue. Thanks for pointing that out!